x-pack/plugins/ml/public/application/routing/routes/new_job/index_or_search.tsx

```diff
@@ -50,6 +50,16 @@ const getDataVisBreadcrumbs = (navigateToPath: NavigateToPath, basePath: string)
   },
 ];
 
+const getExplainLogRateSpikesBreadcrumbs = (navigateToPath: NavigateToPath, basePath: string) => [
+  getBreadcrumbWithUrlForApp('ML_BREADCRUMB', navigateToPath, basePath),
```

Comment:
If I go into the Data Visualizer - Data View, the breadcrumbs now indicate I'm in the AIOps section:

![image](https://user-images.githubusercontent.com/7405507/168557303-fa1e5f30-199b-46cf-9f8a-f2a6135e01bb.png)

and when I go into the new Search to explain page, the Data visualizer breadcrumb is displayed:

![image](https://user-images.githubusercontent.com/7405507/168557516-ba545747-990c-40b9-bc15-0e424923ae7a.png)

Reply:
Fixed the breadcrumbs in [b39c970](https://github.com/elastic/kibana/pull/132121/commits/b39c97041c85cae2b182f3e2765b2a17d83aab14).